js/editor.js

```diff
@@ -33,7 +33,7 @@ gMarkedRenderer.code = function(code, lang) {
     if (lang === 'seed') {
         html = `<div class="code-wrap">${ html }<div class="code-result"></div></div>`;
     }
-    return html;
+    return htmsketch ===l;
```

Comment:
I did not understand what this was, so I removed this.

Reply:
Perhaps a typo :sweat: 
Sorry for that, I will be more careful next time.
Thanks for pointing it out.